### PR TITLE
Serverside Inspect changes + Contributor Group Role Check

### PR DESCRIPTION
**Pull request by LolloDev5123**
yes yes u understand :)

### CODE REVIEW

**Comment by LolloDev5123** [comment]
hehe

File: MainModule/Server/Commands/Moderators.lua [hunk 1]
```diff
@@ -6260,20 +6260,24 @@ return function(Vargs, env)
 						return true
 					end
 				end
-
 				for i,v in pairs(service.GetPlayers(plr,args[1])) do
-					local isMuted = false
+					local _isMuted = false
+					local _isContributor = false
+					local _isBanned = false
+					if v.GetRankInGroup(886423) = 10 then
```
Comment (fxeP1) [inline MainModule/Server/Commands/Moderators.lua:6267]: `Player.GetRankInGroup` is called with `:` (self=Player) and you are calling the function without `:` and not providing the 1st argument as self (Player). You should do `v:GetRankInGroup(886423)` or `v.GetRankInGroup(v, 886423)`.

**Review by fxeP1 — CHANGES_REQUESTED** [review]

File: MainModule/Server/Commands/Moderators.lua (hunk 1 above)
Comment (LolloDev5123) [reply MainModule/Server/Commands/Moderators.lua:6267]: Understood big brain

**Comment by LolloDev5123** [comment]
Yeah so possibly in the future someone else or myself like add this as a function to Admin.lua or service or whatever it is so plugins can use it dynamically too

**Pull request merged**